hudi-spark-datasource/hudi-spark3.2plus-common/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieSpark3Analysis.scala

```diff
@@ -129,50 +129,6 @@ class HoodieSpark3Analysis(sparkSession: SparkSession) extends Rule[LogicalPlan]
   }
 }
 
-/**
- * Rule for resolve hoodie's extended syntax or rewrite some logical plan.
- */
-case class HoodieSpark3ResolveReferences(sparkSession: SparkSession) extends Rule[LogicalPlan]
```

Comment:
can you help clarify why this Rule is no longer needed?

Reply:
also should this file be under spark3-common? or name as HoodieSpark32PlusAnalysis?

Reply:
This rule actually was never used